packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapController.h

```diff
@@ -5,6 +5,7 @@
 #import <Flutter/Flutter.h>
 #import <GoogleMaps/GoogleMaps.h>
 #import "GoogleMapCircleController.h"
+#import "GoogleMapHeatmapController.h"
```

Comment:
Can this import be moved to GoogleMapController.m since it is not used in the header file?

Reply:
Yes